sdk/batch/tests.mgmt.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+pr: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
See `MatrixFilters` in /eng/pipelines/templates/stages/archetype-sdk-tests.yml

Root is PS code so it might be hard to understand.

Some sample found there.
```
      MatrixFilters:
        # Exclusion example
        - OSVmImage=^(?!macOS).*
```

Maybe just use windows?

Reply:
OK.
JobMatrix generated for windows is like:
```
"windows2019_111_netty_False_verify_DskipCompiletrueDskipTestCompiletrueDcreateSourcesJarfalse": {
    "OSVmImage": "MMS2019",
    "Pool": "azsdk-pool-mms-win-2019-general",
    "JavaTestVersion": "1.11",
    "AZURE_TEST_HTTP_CLIENTS": "netty",
    "TestFromSource": false,
    "TestGoals": "verify",
    "TestOptions": "-DskipCompile=true -DskipTestCompile=true -DcreateSourcesJar=false"
  }
```

I'll try
``` 
MatrixFilters:
    - pool=.*(win).*
```

Thanks!